### DISCUSSION TITLE
Logging Title missing in OTEL Log Model

### DISCUSSION THREAD

**Discussion by rober15** [open-telemetry/community#1851] [Community Discussions]
The OTEL Log Model: https://opentelemetry.io/docs/specs/otel/logs/data-model/ only defines for the log entry just a body but not title. 
But for example in google (gcp) you can differ between the log body (json_payload, text_payload, proto_payload) and the title displayed in logging. The title is shown in log explorer in single line and you can expand then entire log when needed.

So when you have a more complex log body (maybe even a structured log type like json) you can set additional a title.
You can find that field in google: https://cloud.google.com/logging/docs/structured-logging
Field: message

Any idea how to influence the title gcp log field: "message" via OTEL Log ?

Many thanks for your help.

**Comment by jack-berg** [comment]
Can you open an issue in https://github.com/open-telemetry/opentelemetry-specification to discuss? Thanks.
